### ISSUE TITLE
Trying to edit ckeditor toolbar

### ISSUE THREAD

**Issue by morais17** [jipiboily/monologue#292]
Hello,
i've been trying to edit ckeditor toolbar to include some plugins about line spacing and height. However i've been unable to edit the toolbar.

I've tried creating ckeditor/config.js and changing there but it doesnt change. I've also tried changing monologue/admin/ckeditor-config.js without sucess .

Is there any file i should be changing that i missed?

Thanks

**Comment by jipiboily** [comment]
When you say you tried changing `monologue/admin/ckeditor-config.js`, did you make a copy in your project at the same exact location? The file should be exactly at this location in your project: `app/assets/javascripts/monologue/admin/ckeditor-config.js`, then it should work.

**Comment by morais17** [comment]
Yes, i did in the same location. I should have specified better with the assets path sorry. I tried removing some options from the toolbar to check if it was changing but it was always the same, nothing was changing :/

**Comment by jipiboily** [comment]
I have no idea why it would not work honestly. Do you have a repo I could look at to figure this out?
